www/main/viewer/utils/chromecast.js

```diff
@@ -1,182 +1,192 @@
-const remote = require("@electron/remote");
+/* global chrome */
+import { ipcRenderer } from 'electron';
+
+const remote = require('@electron/remote');
+
+/* eslint-disable global-require */
+export const tingle = require('../../../assets/js/vendor/tingle');
+/* eslint-enable */
+
 const { i18n } = remote.require('./app');
-export const tingle = require('../../../assets/js/vendor/tingle.js');
 const analytics = remote.getGlobal('analytics');
 
 const applicationID = 'ECF05819';
 const namespace = 'urn:x-cast:com.khalis.cast.sttm.gurbani';
 let session = null;
 let isCastInitialized = false;
 
+/**
+ * append message to debug message window
+ * @param {string} message A message string
+ */
+export const appendMessage = (message) => {
+  /* eslint-disable no-console */
+  console.log(message);
+  /* eslint-enable */
+};
+
+const displayError = (errorMessage) => {
+  const modal = new tingle.Modal({
+    footer: true,
+    stickyFooter: false,
+    closeMethods: ['overlay', 'button', 'escape'],
+  });
+
+  modal.setContent(`<h2>${errorMessage}</h2>`);
+  // add ok button
+  modal.addFooterBtn('OK', 'tingle-btn tingle-btn--pull-right tingle-btn--default', () => {
+    modal.close();
+  });
+
+  // open modal
```

Comment:
Veerji, I think these obvious comments can be removed. However, I think this is a little subjective thing. What do you say?

Reply:
I think this makes it easier to find code, we can search for open modal to see this entire code. Similarly searching for ok button will show us that button code.

Reply:
Okay veerji. I was thinking these comments are too obvious for this piece of code. However, we can keep this as it is to find code easily.